src/config/consts.js

```diff
@@ -22,6 +22,8 @@ export const consts = {
 
   MATOMO_TRACKING: {
     SCREEN_VIEW: {
+      CONSTRUCTION_SITE_DETAIL: 'Construction site detail',
+      CONSTRUCTION_SITE_OVERVIEW: 'Construction site overview',
```

Comment:
i think we should avoid "overview" and "detail", as we do not have that for other pages. maybe we can use some plural like "Constructions"? maybe we need some namespace like "Widget / ..." as well. was this string already discussed somehow? if not, we should add it to the meeting agenda. 

please add it to the matomo doc in confluence at the end.

Reply:
we can discuss it in the meeting, as it was not discussed at any other place yet.

I will add it to the confluence site at the end.

Reply:
<img width="455" alt="Bildschirmfoto 2021-01-12 um 12 53 43" src="https://user-images.githubusercontent.com/1942953/104311264-36421600-54d5-11eb-88bd-ebfb410c1661.png">

Reply:
:+1: